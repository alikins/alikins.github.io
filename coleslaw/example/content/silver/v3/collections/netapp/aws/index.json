{
    "name": "aws",
    "namespace": "netapp",
    "deprecated": false,
    "href": "example/content/silver/v3/collections/netapp/aws/",
    "versions_url": "example/content/silver/v3/collections/netapp/aws/versions/",
    "highest_version": {
        "version": "20.2.0",
        "href": "example/content/silver/v3/collections/netapp/aws/versions/20.2.0/"
    }
}