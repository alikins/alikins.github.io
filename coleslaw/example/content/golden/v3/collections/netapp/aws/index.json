{
    "name": "aws",
    "namespace": "netapp",
    "deprecated": false,
    "href": "example/content/golden/v3/collections/netapp/aws/",
    "versions_url": "example/content/golden/v3/collections/netapp/aws/versions/",
    "highest_version": {
        "version": "20.2.0",
        "href": "example/content/golden/v3/collections/netapp/aws/versions/20.2.0/"
    }
}